src/app/features/reporter/reporter-dialog.component.html

```diff
@@ -66,5 +66,5 @@
 </div>
 <systelab-dialog-bottom>
 	<button type="button" class="btn mr-0" (click)="doUpdateTestCase()">Update Test Case</button>
-	<button type="button" class="btn ml-auto" (click)="doRun()">Report</button>
+	<button type="button" class="btn ml-auto" [disabled]="!isValidForm()" (click)="doRun()">Report</button>
```

Comment:
The conditions to disable this button must be Project, Test Plan, Test Cycle, Test Group and New Test Cycle name. All of them must be met. I suggest to change isValidForm to include all of them.

Reply:
I think the method is already correct, what could we add more?
```
	public isValidForm() {
		if (this._userId) {
			if (this.selectedTestCycleId) {
				return true;
			} else {
				return this.selectedProjectId && this.selectedTestPlanId && this.selectedTestGroups.length > 0 && this.nameForNewTestCycle !== '';
			}
		} else {
			return false;
		}
	}
```

Reply:
Both buttons must be disabled on this case 
![image](https://user-images.githubusercontent.com/1970034/128498219-7b6ad5d7-55a4-46a6-9613-52a153124532.png)

The condition if (this.selectedTestCycleId) { return true; } must be removed and include all the fields in just one